src/test/config.sol

```diff
@@ -957,7 +957,7 @@ contract Config {
             offboarding:  false
         });
         afterSpell.collaterals["LSEV2-SKY-A"] = CollateralValues({
-            aL_enabled:   true,
+            aL_enabled:   false,
             aL_line:      50 * MILLION,
             aL_gap:       25 * MILLION,
             aL_ttl:       24 hours,
```

Comment:
We should probably remove the AutoLine parameters from here since the item is being removed from it.